thirtyinch-rx2/build.gradle

```diff
@@ -23,6 +24,7 @@ android {
 
 dependencies {
     implementation project(':thirtyinch')
+    api "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
```

Comment:
🙅‍♂️
Even if I agree with you, that Android projects starts more and more in Kotlin we should keep this to pure Java as long as we haven't discussed that internally first.

Reply:
Good point. Then we have to probably put this PR on hold until it has been decided that we allow Kotlin code in ThirtyInch after all.